heron/api/tests/java/BUILD

```diff
@@ -3,6 +3,7 @@ load("/tools/rules/heron_deps", "heron_java_api_proto_files")
 api_deps_files = [
     "//heron/api/src/java:api-java",
     "//heron/common/src/java:utils-java",
+    "//heron/common/src/java:basics-java",
```

Comment:
why are we adding this dep?

Reply:
com.twitter.heron.common.basics.ByteAmount is from there